Build/Scripts/runTests.sh

```diff
@@ -631,8 +661,12 @@ if [[ ${IS_CORE_CI} -eq 1 ]]; then
 else
     echo "Environment: local" >&2
 fi
-echo "PHP: ${PHP_VERSION}" >&2
-echo "TYPO3: ${CORE_VERSION}" >&2
+if [[ ${TEST_SUITE} =~ ^npm$ ]]; then
+    echo "NODE: ${NODE_VERSION}" >&2
```

Comment:
For both commands, I still see the PHP and TYPO3 versions:

```
Now running 'npm ci --silent'.

> tea@1.0.0 fix:lint:js
> eslint 'Resources/Public/**/*.js' --quiet --fix


###########################################################################
Result of lintJs
Environment: local
PHP: 8.3
TYPO3: 12.4
CONTAINER_BIN: podman
SUCCESS
###########################################################################
```

This doesn't look intentional to me.